src/main/java/com/endava/soa_5/exception_and_io/SetProperty.java

```diff
@@ -0,0 +1,41 @@
+package com.endava.soa_5.exception_and_io;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class SetProperty {
+    private String result;
+    private InputStream inputStream;
+
+    public String getPropValues(String key) throws IOException {
+
+        try {
+            result = new String();
```

Comment:
Put this initializations inside a default (or paametrized) constructor.

Reply:
Done!